src/heat_balance.jl

```diff
@@ -2,9 +2,68 @@
 # Heat balance
 
 
-function heat_balance() end
+function energy_balance() end
```

Comment:
how about we keep it as 'heat_balance' because it is a distinct aspect of energy balance

Reply:
yes!